docs/explanation/rocks.rst

```diff
@@ -3,22 +3,62 @@
 ROCKs
 =====
 
-ROCKs are container images that are compliant with the `Open Container
-Initiative`_ (OCI) `image format specification <OCI_image_spec_>`_.
-This means that ROCKs are interoperable with other tools that work with
-OCI-compliant images.
+ROCKs are container images based on Ubuntu designed to meet
+cloud-native software's security, stability, and reliability requirements.
+
+ROCKs were created to address two primary needs. Firstly, it tackles
+the current issues with container image security, as many popular
+container images often have known vulnerabilities.
+Secondly, it aims to provide a smaller container image based on
+Ubuntu to reduce the resource footprint on the host.
 
-A ROCK can be stored in an existing container registry, very much like any
-Docker image, and run in the same way as any other container image.
-For example, Docker can be used to run a ROCK with the usual command line
-syntax:
+These container images comply with the `Open Container
+Initiative`_ (OCI) `image format specification <OCI_image_spec_>`_.
+They can be stored in container registries (e.g. DockerHub, ECR, ACR, Zot,..)
```

Comment:
why Zot? sounds like an odd reference out of nowhere...

Reply:
Yeah true, Zot is not a big actor on container registry also